readthedocs/templates/wipe_version.html

```diff
@@ -1,16 +1,31 @@
 {% extends "base.html" %}
 
+{% block project_editing %}
+  {% with versions_active="active" %}
+    {% include "core/project_bar.html" %}
+  {% endwith %}
+{% endblock %}
+
+
 {% block title %}Wipe the build directories for a version {% endblock %}
 
 {% block content %}
 
+{% blocktrans with slug=version.slug %}
+    <h3>Remove build environment for <em>{{ slug }}</em> version.</h3>
+{% endblocktrans %}
+
 {% if deleted %}
-Your project environment has been wiped.<br><br>
+    {% blocktrans %}
+        Your project environment has been wiped.<br><br>
```

Comment:
All `<br>` can be removed. Spacing should be forced with a surrounding `p` element, or other css.